lucien-ios/Add New Comic Book/AddComicViewController.swift

```diff
@@ -0,0 +1,208 @@
+
+        // Issue
+        cconfigureLabelwith(label: issueLabel, fontName: LucienTheme.Fonts.Muli.light.rawValue, size: 12, color: LucienTheme.coolGrey)
+        configureTextFieldBorder(textField: issueTextField)
+        issueTextField.attributedPlaceholder = createPlaceHolderAttributedString(textFieldPlaceholderText: "Issue Number")
+
+        // Publisher
+        cconfigureLabelwith(label: publisherLabel, fontName: LucienTheme.Fonts.Muli.light.rawValue, size: 12, color: LucienTheme.coolGrey)
+        configureTextFieldBorder(textField: publisherTextField)
+        publisherTextField.attributedPlaceholder = createPlaceHolderAttributedString(textFieldPlaceholderText: "Marvel, DC Comics, Image, Dark Horse…")
+
+        // Release Date
+        cconfigureLabelwith(label: releaseDateLabel, fontName: LucienTheme.Fonts.Muli.light.rawValue, size: 12, color: LucienTheme.coolGrey)
+        configureUIButton(button: selectADateButton)
+
+        // Genre
+        cconfigureLabelwith(label: genreLabel, fontName: LucienTheme.Fonts.Muli.light.rawValue, size: 12, color: LucienTheme.coolGrey)
+        configureUIButton(button: selectAGenreButton)
+
+        // Condition
+        cconfigureLabelwith(label: conditionLabel, fontName: LucienTheme.Fonts.Muli.light.rawValue, size: 12, color: LucienTheme.coolGrey)
+        configureUIButton(button: selectAConditionButton)
+    }
+
+    private func cconfigureLabelwith(label: UILabel, fontName: String, size: CGFloat, color: UIColor) {
+        label.font = UIFont(name: fontName, size: size)
+        label.textColor = color
+    }
+
+    private func configureTextFieldBorder(textField: UITextField) {
+        let bottomBorder = CALayer()
+        bottomBorder.frame = CGRect(x: 0.0, y: textField.frame.height - 3.0, width: textField.frame.width, height: 1.0)
+        bottomBorder.backgroundColor = LucienTheme.silver.cgColor
+        textField.borderStyle = UITextBorderStyle.none
+        textField.layer.addSublayer(bottomBorder)
+    }
+
+    private func configureUIButton(button: UIButton) {
+        button.setImage(UIImage(named: "dropDownArrow"), for: .normal)
+        button.semanticContentAttribute = .forceRightToLeft
+        button.imageEdgeInsets.left = button.frame.width - 24
+        button.tintColor = LucienTheme.dark
+    }
+
+    private func createPlaceHolderAttributedString(textFieldPlaceholderText: String) -> NSAttributedString {
+        let placeholderAttributes = [
+        NSAttributedStringKey.foregroundColor: LucienTheme.silver,
+        NSAttributedStringKey.font : UIFont(name: LucienTheme.Fonts.Muli.regular.rawValue, size: 16) ?? UIFont()
+        ] as [NSAttributedStringKey : Any]
+
+       return NSAttributedString(string: textFieldPlaceholderText, attributes: placeholderAttributes)
+    }
+
+    // MARK: - IBOutlet Methods
+
+    @IBAction func selectADateButtonTapped(_ sender: UIButton) {
+        UIView.animate(withDuration: 0.3, animations: { [weak self] in
```

Comment:
Whenever possible, try to use trailing closures. Also, there's no risk for a retain cycle in `UIView.animate` so you don't need to capture self weakly e.g.
```
UIView.animate(withDuration: 0.3) {
    self.datePicker.isHidden = false
}
```